libs/sdk-ui-filters/src/AttributeFilter@next/Components/Addons/AttributeDisplayFormSelect/AttributeDisplayFormDropdownItem.tsx

```diff
@@ -0,0 +1,71 @@
+// (C) 2021-2022 GoodData Corporation
+//TODO delete it moved to SDK
+import React from "react";
+import classNames from "classnames";
+import { stringUtils } from "@gooddata/util";
+import { IAttributeDisplayFormMetadataObject, ObjRef } from "@gooddata/sdk-model";
+import { ShortenedText } from "@gooddata/sdk-ui-kit";
+
+enum AttributeDisplayFormType {
+    HYPERLINK = "GDC.link",
+    GEO_PUSHPIN = "GDC.geo.pin",
+}
+
+const getDisplayFormIcon = (type: string) => {
+    switch (type) {
+        case AttributeDisplayFormType.HYPERLINK:
+            return "gd-icon-hyperlink-warning";
+        case AttributeDisplayFormType.GEO_PUSHPIN:
+            return "gd-icon-earth";
+        default:
+            return "gd-icon-label-warning";
+    }
+};
+
+const tooltipAlignPoints = [
+    { align: "cl cr", offset: { x: -10, y: 0 } },
+    { align: "cr cl", offset: { x: 10, y: 0 } },
+];
+
+/**
+ * @internal
+ */
+export interface IDisplayFormDropdownItemProps {
+    displayForm: IAttributeDisplayFormMetadataObject;
+    onClick: (displayForm: ObjRef) => void;
+    selected: boolean;
+}
+
+/**
+ * @internal
+ */
+export const AttributeDisplayFormDropdownItem: React.FC<IDisplayFormDropdownItemProps> = (props) => {
```

Comment:
Component name + props name should be aligned eg `AttributeDisplayFormSelectItem` `IAttributeDisplayFormSelectItemProps`